ArcGISRuntimeSDKQt_QMLSamples/Geometry/NearestVertex/NearestVertex.qml

```diff
@@ -120,7 +120,6 @@ Rectangle {
     }
 
     MouseArea {
-        anchors.fill: parent
```

Comment:
this allows you to pan, but does the onClicked signal ever get called now? Presumably, this all should be be moved into a onMouseClicked signal handler on MapView instead of setting up a separate MouseArea